ux-of-data-fetching/Exercise1/solution/styles.css

```diff
@@ -0,0 +1,87 @@
+:root {
+  font-family: Inter, system-ui, Avenir, Helvetica, Arial, sans-serif;
+  line-height: 1.5;
+  font-weight: 400;
+}
+
+#root {
+  max-width: 1280px;
+  margin: 0 auto;
+  padding: 2rem;
+  text-align: center;
+  display: flex;
+  flex-direction: column;
+}
+
+body {
+  margin: 0;
+  display: flex;
+  min-width: 320px;
+  min-height: 100vh;
+}
+
+table {
+  border-collapse: collapse;
+  border-spacing: 0;
+  border: 1px solid #eee;
+}
+
+tr {
+  border-bottom: 1px solid #eee;
+  height: 40px;
+}
+tr:nth-child(even) {
+  background-color: #fafafa;
+}
+
+th,
+td {
+  padding: 0 20px;
+}
+
+th:first-child,
+td:first-child {
+  width: 120px;
+  text-align: left;
+}
+
+th:nth-child(2),
+td:nth-child(2) {
+  width: 50px;
+  text-align: right;
+}
+
+th:nth-child(3),
+td:nth-child(3) {
+  width: 180px;
+  text-align: left;
+}
+
+button {
+  border: 0;
+  color: white;
+  background-color: #13b0ff;
+  padding: 10px 20px;
+  border-radius: 4px;
+  cursor: pointer;
+  transition: background-color 0.2s ease;
+  margin: 10px;
+}
+button:hover {
+  background-color: #1199e6;
+}
+button:focus {
+  outline: none;
```

Comment:
No focus indicator?

Reply:
lol not sure why i have this in here, got rid of it